src/Map/src/Map.php

```diff
@@ -159,23 +296,23 @@ public static function fromArray(array $map): self
             $map['fitBoundsToMarkers'] = false;
         }
 
-        $map['markers'] ??= [];
+        $map['markers'] ??= new SplObjectStorage();
```

Comment:
Should be reverted, `$map['markers']` has not the same goal than `Map::$markers`. Same thing for similar code below